### PR TITLE
tools: add specific operations in TiDB-Binlog Cluster

### PR DESCRIPTION
**Pull request by KASSADAR**
修正 TiDB-Binlog Cluster 版本用户文档 中 pump 的部署操作 以及 Drainer 以 PB 作为输出的一个小BUG

增加具体的操作场景，分为在已有集群中部署 和 从零开始部署。列出在已有集群中部署的操作步骤。

### CODE REVIEW

**Comment by lilin90** [comment]
@KASSADAR Please sign the CLA. Thanks!

File: tools/mydumper.md [hunk 1]
```diff
@@ -33,6 +35,10 @@ category: tools
 ./bin/mydumper -h 127.0.0.1 -u root -P 4000
 ```
 
+```bash
+./mydumper -h 127.0.0.1 -P 3306 -u backup -p cJebJ4aBmS8H -t 16 -F 64 --regex '^(?!(mysql|information_schema|sys|performance_schema))' --skip-tz-utc -o /tmp/mysql_data
```
Comment (WangXiangUSTC) [inline tools/mydumper.md]: please use a simple example

Comment (WangXiangUSTC) [reply tools/mydumper.md]: and add the description about  this command

**Comment by GregoryIan** [comment]
thanks @KASSADAR，  @WangXiangUSTC PTAL

File: tools/tidb-binlog-cluster.md [hunk 2]
```diff
@@ -237,7 +252,7 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
         # Uncomment this if you want to use `pb` or `sql` as `db-type`.
         # `Compress` compresses the output file, like the `pb` and `sql` file. Now it supports the `gzip` algorithm only. 
         # The value can be `gzip`. Leave it empty to disable compression. 
-        [syncer.to]
+        #[syncer.to]
```
Comment (WangXiangUSTC) [inline tools/tidb-binlog-cluster.md]: don't modify this line, and remove the comment at line 258

File: tools/tidb-binlog-cluster.md [hunk 3]
```diff
@@ -361,7 +376,7 @@ Drainer="192.168.0.13"
         ```
   
         如果命令行参数与配置文件中的参数重合，则使用命令行设置的参数的值。
-
+
```
Comment (WangXiangUSTC) [inline tools/tidb-binlog-cluster.md]: have blank

File: tools/tidb-binlog-cluster.md [hunk 4]
```diff
@@ -132,9 +132,24 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
...
+    （a）在已有的 TiDB 集群上增加 Pump 组件，需按以下步骤逐步进行
+         
+         // 1）部署 pump_servers 和 node_exporters
+         ansible-playbook deploy.yml -l pump_servers,monitored_servers
+         
+         // 2）更新并重启 tidb_servers
+         ansible-playbook rolling_update.yml --tags=tidb
+         
+         // 3）更新监控信息
+         ansible-playbook rolling_update_monitor.yml
```
Comment (WangXiangUSTC) [inline tools/tidb-binlog-cluster.md]: @liubo0127 PTAL

File: tools/mydumper.md [hunk 5]
```diff
@@ -24,6 +24,8 @@ category: tools
                        默认值：当前 TSO（SHOW MASTER STATUS 输出的 UniqueID）
                        此参数可设为 TSO 时间或有效的 datetime 时间。例如：-z "2016-10-08 16:45:26"
 ```
+### 需要的权限
```
Comment (lilin90) [inline tools/mydumper.md]: Please add a blank line both before and after the heading line.

File: tools/mydumper.md [hunk 6]
```diff
@@ -24,6 +24,8 @@ category: tools
...
+SELECT, RELOAD, LOCK TABLES, REPLICATION CLIENT
```
Comment (lilin90) [inline tools/mydumper.md]:
```suggestion
- SELECT
- RELOAD
- LOCK TABLES
- REPLICATION CLIENT
```

File: tools/tidb-binlog-cluster.md [hunk 7]
```diff
@@ -132,9 +132,24 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
         pump3 ansible_host=172.16.10.74 deploy_dir=/data1/pump
         ```
 
-2. 部署并启动 TiDB 集群
-
-    使用 Ansible 部署 TiDB 集群的具体方法参考 [TiDB Ansible 部署方案](../op-guide/ansible-deployment.md)，开启 binlog 后默认会部署和启动 Pump 服务。
+2. 部署并启动含 pump 组件的 TiDB 集群
```
Comment (lilin90) [inline tools/tidb-binlog-cluster.md]:
```suggestion
2. 部署并启动含 Pump 组件的 TiDB 集群
```

File: tools/tidb-binlog-cluster.md [hunk 8]
```diff
@@ -132,9 +132,24 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
...
+    
+    参照上文配置完 inventory.ini 后，选择一种方式进行部署
```
Comment (lilin90) [inline tools/tidb-binlog-cluster.md]:
```suggestion
    参照上文配置完 `inventory.ini` 文件后，从以下两种方式中选择一种进行部署。
```

File: tools/tidb-binlog-cluster.md [hunk 9]
```diff
@@ -132,9 +132,24 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
...
+    
+    （a）在已有的 TiDB 集群上增加 Pump 组件，需按以下步骤逐步进行
+         
+         // 1）部署 pump_servers 和 node_exporters
```
Comment (lilin90) [inline tools/tidb-binlog-cluster.md]: - The "//" characters in this part make it like a code block, which is weird. Please remove them.
- Use inline code format or code block format only for code.

File: tools/tidb-binlog-cluster.md [hunk 10]
```diff
@@ -132,9 +132,24 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
...
+         ansible-playbook deploy.yml -l pump_servers,monitored_servers
+         
+         // 2）更新并重启 tidb_servers
+         ansible-playbook rolling_update.yml --tags=tidb
+         
+         // 3）更新监控信息
+         ansible-playbook rolling_update_monitor.yml
+
+    
+    （b）从零开始部署含 pump 组件的 TiDB 集群
```
Comment (lilin90) [inline tools/tidb-binlog-cluster.md]:
```suggestion
    （b）从零开始部署含 Pump 组件的 TiDB 集群
```

File: tools/tidb-binlog-cluster.md [hunk 11]
```diff
@@ -163,7 +178,7 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
     2018/06/21 11:24:47 meta.go:117: [info] meta: &{CommitTS:400962745252184065}
     ```
 
-    该命令会生成一个文件 `{data-dir}/savepoint`，该文件中包含 tso，用该 tso 作为 Drainer 初次启动使用的 `initial-commit-ts` 参数的值。
+    该命令会输出 meta: &{CommitTS:400962745252184065}，CommitTS 的值作为 Drainer 初次启动使用的 `initial-commit-ts` 参数的值。
```
Comment (lilin90) [inline tools/tidb-binlog-cluster.md]:
```suggestion
    该命令会输出 `meta: &{CommitTS:400962745252184065}`，CommitTS 的值作为 Drainer 初次启动使用的 `initial-commit-ts` 参数的值。
```

**Comment by lilin90** [comment]
@liubo0127 PTAL Thanks!

File: tools/tidb-binlog-cluster.md [hunk 12]
```diff
@@ -132,9 +132,33 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
...
+2. 部署并启动含 Pump 组件的 TiDB 集群
+
+    参照上文配置完 `inventory.ini` 文件后，从以下两种方式中选择一种进行部署。
+    
+    **方式一**：在已有的 TiDB 集群上增加 Pump 组件，需按以下步骤逐步进行。
+    
+    1. 部署 pump_servers 和 node_exporters
+    
+        ```
+        ansible-playbook deploy.yml -l pump_servers,monitored_servers
```
Comment (liubo0127) [inline tools/tidb-binlog-cluster.md]: This operation is a bit heavy, changed to `ansible-playbook deploy.yml -l ${pump1_ip}, ${pump2_ip}` is better

File: tools/tidb-binlog-cluster.md [hunk 13]
```diff
@@ -132,9 +132,33 @@ Pump 和 Drainer 都支持部署和运行在 Intel x86-64 架构的 64 位通用
...
+    2. 更新并重启 tidb_servers
+    
+        ```
+        ansible-playbook rolling_update.yml --tags=tidb
+        ```
+        
+    3. 更新监控信息
+    
+        ```
+        ansible-playbook rolling_update_monitor.yml
```
Comment (liubo0127) [inline tools/tidb-binlog-cluster.md]: Just update the prometheus configuration: `ansible-playbook rolling_update_monitor.yml --tags=prometheus`

**Comment by liubo0127** [comment]
LGTM

**Pull request merged**